.github/workflows/test.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     - name: Install pnpm and dependencies
       uses: pnpm/action-setup@v2
       with:
-        version: 7
+        version: 8
```

Comment:
I made changes to fix the versions in our workflows in [this PR](https://github.com/warp-ds/vue/pull/44).